docs/spaces/index.asciidoc

```diff
@@ -108,6 +109,12 @@ interface.
 {kib} also has beta <<saved-objects-api-import, import>> and 
 <<saved-objects-api-export, export>> APIs if you want to automate this process.
 
+[float]
+[[spaces-default-route]]
+=== Space-level landing page
+
+You can create a custom experience for users by configuring the {kib} landing page on a per-space basis.
```

Comment:
Add this sentence after the first sentence:

The landing page can route users to a specific dashboard, application, or saved object as they enter each space.